runners/flink/flink_runner.gradle

```diff
@@ -169,6 +170,17 @@ dependencies {
   validatesRunner project(path: ":runners:core-java", configuration: "testRuntime")
   validatesRunner project(project.path)
   miniCluster "org.apache.flink:flink-runtime-web_2.11:$flink_version"
+  compile project(path: ":model:fn-execution", configuration: "shadow")
+  compile project(path: ":model:pipeline", configuration: "shadow")
+  compile project(path: ":model:job-management:", configuration: "shadow")
+  compile project(":sdks:java:fn-execution")
+  compile project(path: ":vendor:sdks-java-extensions-protobuf", configuration: "shadow")
+  permitUnusedDeclared library.java.jackson_annotations
```

Comment:
Why does `jackson_annotations` need to be declared? Can we remove it?

Reply:
We cannot remove jackson_annotations from here. As it is transitively being declared here but not being used.

Reply:
I see `compile library.java.jackson_annotations` was removed. So there's no need to have `permitUnusedDeclared library.java.jackson_annotations` anymore because we are not declaring it (I don't think the plugin checks for "transitive declaration," either we explicitly declare a dependency or we don't declare it at all).

Reply:
Now, we can safely remove it. removed, thanks!